invenio_app_ils/ill/serializers/custom_fields.py

```diff
@@ -6,19 +6,18 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Loan custom serializer functions."""
-
+from invenio_circulation.proxies import current_circulation
```

Comment:
super nit: not even a pep error but every other file has an empty line after the docstring